### PR TITLE
Added info on breaking changes to release notes for 1.6.0

### PR DESCRIPTION
**Pull request by dmccoystephenson**
## Problem
Programs that rely on the output of the ODE will experience issues when updating to 1.6.0, and there is no mention of this in the release notes.

## Solution
Information on breaking changes has been added to the release notes.

### CODE REVIEW

File: docs/Release_notes.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ Enhancements in this release:
 Known Issues:
 - No known issues at this time.
 
+Breaking Changes:
+- Users should note that due to the switch to J2735 2020, some fields in outputted TIMs will be different. Any programs relying on data from the ODE must be modified to accommodate these updated outputs.
```
Comment (drewjj) [inline docs/Release_notes.md:19]: Lets specify the relevant fields here or a link to a place noting the differences for convenience. Not everyone is an expert of the J2735.

**Review by drewjj — CHANGES_REQUESTED** [review]

File: docs/Release_notes.md (hunk 1 above)
Comment (dmccoystephenson) [reply docs/Release_notes.md]: Good call! The relevant fields & their new names have been added to this section.

**Review by drewjj — APPROVED** [review]

File: docs/Release_notes.md (hunk 1 above)
Comment (drewjj) [reply docs/Release_notes.md]: Awesome

**Review by drewjj — COMMENTED** [review]
Looks good. The build just inconsistently passes I guess. and I see no reason why this would happen. Oh well. ¯\_(ツ)_/¯

**Review by drewjj — APPROVED** [review]

**Review by dan-du-car — APPROVED** [review]

**Pull request merged**